### DISCUSSION TITLE
这个是对的吗？

### DISCUSSION THREAD

**Discussion by jian-li-5205** [share121/inter-knot#2708] [其他]
![1723969377368](https://github.com/user-attachments/assets/262e9c6f-ff7a-4f58-b1ba-96025f101a33)

**Comment by homteam** [comment]
看着像，网址是不是：https://store.steampowered.com/about/

**Reply by jian-li-5205** [reply]
原来是这个，安装好了

**Reply by homteam** [reply]
人机验证等着你👍

**Reply by jian-li-5205** [reply]
那这是怎么回事
![358893374-374dafaa-6867-4345-9fc0-16bc61dc27fb](https://github.com/user-attachments/assets/6cee9ca4-b387-46e9-880b-1e2f5ee79b6c)

**Reply by isle200inf** [reply]
我勒个烧钢啊你是真的假的

**Reply by lllliufeng** [reply]
我去，遇到真的了

**Reply by isle200inf** [reply]
## 他绳网都上来了，不能是真的吧

**Reply by lllliufeng** [reply]
刷到过这个，原版好像是剧本（

**Reply by isle200inf** [reply]
<img src="https://img-static.mihoyo.com/communityweb/upload/6744fdc365a234f152bc3fcc6443c7a9.png" alt="米游姬糖葫芦" width="75" >

**Reply by LeiMoo** [reply]
> 看着像，网址是不是：https://store.steampowered.com/about/

s.team这个短域名好记一些

**Reply by shan-dgha** [reply]
> 那这是怎么回事 ![358893374-374dafaa-6867-4345-9fc0-16bc61dc27fb](https://private-user-images.githubusercontent.com/173781266/358893809-6cee9ca4-b387-46e9-880b-1e2f5ee79b6c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM5ODYzNzMsIm5iZiI6MTcyMzk4NjA3MywicGF0aCI6Ii8xNzM3ODEyNjYvMzU4ODkzODA5LTZjZWU5Y2E0LWIzODctNDZlOS04ODBiLTFlMmY1ZWU3OWI2Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxOFQxMzAxMTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yN2Q1OWU4NjcyZmY0MzViMTRkN2IxZTI2NmIyOGM4YTA5YWM4NzFiNWJjOTVkNGI4ZjhhNzRhMWMyMTg3YjA5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YKEp2DJtehBxg5l4DDsxpzHHo-v2nEnEsNpP_aOW-xw)

真的假的，这不是网上的一个段子吗？让我遇到了？

**Reply by guanyq000** [reply]
不像是演的

**Reply by zzyuJPG** [reply]
可以下载steam++加速steam和人机验证来完成这部分内容

**Comment by ASTOU2311** [comment]
喔 这位绳网用户，你成功找到
了steam平台
![312123](https://github.com/user-attachments/assets/5edf3397-3d15-4a4b-b811-f9c6aea168c9)

**Comment by qingyangprince** [comment]
这是国内版
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Reply by qingyangprince** [reply]
https://store.steampowered.com/
这个是外服

**Comment by LilangAlan** [comment]
这个委托我接了
